src/scss/_base.scss

```diff
@@ -18,6 +18,14 @@
 		margin: 0;
 		padding: 0;
 	}
+	
+	th {
+		color: oColorsGetColorFor(o-table-header, color);
```

Comment:
Could this and L27 be updated to use `@include oColorsFor` instead of specifying the property?